mlflow/protos/databricks_uc_registry_messages.proto

```diff
@@ -79,6 +79,9 @@ message ModelVersion {
 
   // Details on current ``status``, if it is pending or failed.
   optional string status_message = 12;
+
+  // Current storage location of model version files, e.g. in blob storage
+  optional string storage_location = 13;
```

Comment:
Including changes that have since been made in the backend here. We include this field in the model version response to avoid a redundant call to fetch its storage location during model version creation